api/views/workflow.py

```diff
@@ -228,3 +226,23 @@ def post(self, request, *args, **kwargs):
             return Response({'task_id': async_result.id}, status=status.HTTP_202_ACCEPTED)
         else:
             return Response({'errors': serializer.errors, 'data': prelim_data}, status=status.HTTP_400_BAD_REQUEST)
+
+    def _get_source_config(self, user):
+        config_label = '{}.v1_push'.format(user.username)
+        try:
+            return SourceConfig.objects.get(label=config_label)
```

Comment:
Yeah, that makes sense. Is there already a way to authorize people to use v1 (where the source and config should be created)?